build/Targets/Versions.props

```diff
@@ -14,7 +14,7 @@
     <!-- The release moniker for our packages.  Developers should use "dev" and official builds pick the branch
          moniker listed below -->
     <RoslynNuGetMoniker Condition="'$(RoslynNuGetMoniker)' == ''">dev</RoslynNuGetMoniker>
-    <RoslynNuGetMoniker Condition="'$(OfficialBuild)' == 'true'">beta3</RoslynNuGetMoniker> 
+    <RoslynNuGetMoniker Condition="'$(OfficialBuild)' == 'true'">beta4</RoslynNuGetMoniker> 
```

Comment:
Since this is our last branching for a minor version, we should be resetting this to beta1, and bumping the "2.7" numbers to 2.8.